src/pybind/mgr/dashboard_v2/frontend/src/app/shared/components/view-cache/view-cache.component.html

```diff
@@ -0,0 +1,14 @@
+<alert type="info"
+       *ngIf="status === vcs.ValueNone">
+  Still working, please wait.
```

Comment:
Why not something like this: `We're still trying to get your data, please wait`. @LenzGr you're the expert here

Reply:
done.